README.md

```diff
@@ -118,6 +132,10 @@ Once you've found the appropriate packages for your distribution, run the
 
 ## Testing it out
 
+First you need to edit _~/.local/share/hawck/cfg.lua_ with your keyboard layout
+(eg "us" "uk" etc), as the default is currently hardcoded to "no" (norwegian).
```

Comment:
Instead of

> ... as the default is currently hardcoded to "no" (norwegian) ...

I think the whole paragraph should be replaced with

> As Hawck is not able to determine your Wayland/X11 keymap dynamically, you must first set your keymap in `~/.local/share/hawck/cfg.lua`. This also applies to us-keymap users, as the default is stubbornly set to the more uncommon "no" (norwegian) because the author of Hawck is from there. [FAQ: Keyboard Layout](#faq-layout)

```lua
  keymap = "no", -- set to "us", "de", "no-latin1", etc. depending on your keyboard layout
```

And before the "Is my keyboard layout supported?" headline put, `#### <a name="faq-layout"></a>Is my keyboard lay...` see https://stackoverflow.com/questions/6695439/how-to-link-to-a-named-anchor-in-multimarkdown